docs/source/overview/cli.md

```diff
@@ -0,0 +1,499 @@
+
+### Model architecture configuration
+
+```YAML
+model: {
+    path: /absolute/path/to/file_or_model,
+    class: ModelArchitectureClassName,
+    class_args: {
+        arg1: val1,
+        arg2: val2,
+        ...
+    },
+    non_strand_specific: mean
+}
+```
+
+- `path`: This can be the path to a Python file or a Python module (directory). See the [previous section](#Expected-input-class-and-methods) for details.
+- `class`: The model architecture class name.
+- `class_args`: The arguments needed to instantiate the class.
+- `non_strand_specific`: Optional, possible values are `mean` or `max` if you want to use this parameter. (Otherwise, do not use this key in your model configuration.) If your model does not need to train on strand-specific input sequences, we have implemented a class that will pass both the forward and reverse sequence to the model and either take the `mean` or the `max` value across the two sets of predictions for a sample. 
+
+## A note for the following sections
+For training, evaluation, and analysis [of sequences using trained models], Selene requires that specific keys in the YAML file correspond to function-type values. The value is used to construct an object that is a class in `selene_sdk`. Our [documentation website](http://selene.flatironinstitute.org/) is an important resource for debugging configuration-related errors when you run Selene via the CLI. 
+
+We have tried to cover as many of the common configurations as we can in this document. 
+
+## Train
+An example configuration for training:
+
+```YAML
+train_model: !obj:selene_sdk.TrainModel {
+    batch_size: 64,
+    max_steps: 960000,
+    report_stats_every_n_steps: 32000,
+    save_checkpoint_every_n_steps: 1000,
+    n_validation_samples: 64000,
+    n_test_samples: 960000,
+    cpu_n_threads: 32,
+    use_cuda: True,
+    data_parallel: True,
+    logging_verbosity: 2,
+    checkpoint_resume: False
+}
+```
+
+#### Required parameters
+- `batch_size`:  Number of samples in one forward/backward pass (a single step).
+- `max_steps`: Total number of steps for which to train the model. 
+- `report_stats_every_n_steps`: The frequency with which to report summary statistics. 
+
+#### Optional parameters
+- `save_checkpoint_every_n_steps`: Default is 1000. The number of steps before Selene saves a new checkpoint model weights file. If this parameter is set to `None`, we will set it to the same value as `report_stats_every_n_steps`.
+- `n_validation_samples`: Default is `None`. Specify the number of validation samples in the validation set. If `None`
+   - and the data sampler you use is of type `selene_sdk.samplers.OnlineSampler`, we will by default retrieve 32000 validation samples.
+   - and you are using a `selene_sdk.samplers.MultiFileSampler`, we will use all the validation samples available in the appropriate data file.
+- `n_test_samples`: Default is `None`. Specify the number of test samples in the test set. If `None`
```

Comment:
Since you have sampler info later, it is a bit confusing on how the test sampler will work. Maybe add a link that would bring the reader to the bottom of the page? 
I guess I would add here that if you do not have a test partition, you will train on everything? (if that is true)